### PR TITLE
Adding a community or feature request template

### PR DESCRIPTION
**Pull request by vidya-mohan**
Specific template that builders can use to propose feature requests for the SDK.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation-request.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ---
 name: Documentation request
-about: Need help using the SDK? Spot an error? Let us know how our documentation can be improved.
+about: Need help using the SDK? Spot an error? Let us know how our documentation can
+  be improved.
```
Comment (jing-chen1) [inline .github/ISSUE_TEMPLATE/documentation-request.md:4]: [nit] Do we need a new line here?

**Review by jing-chen1 — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
```
Comment (devalevenkatesh) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]: The other bold questions and the extra line information below them looks more close than this one. So, should we say "A clear and concise description of the solution you like." here?

File: .github/ISSUE_TEMPLATE/community-request-template.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+* Please vote for this issue by adding a 👍 reaction to the issue. This will help the community and maintainers from the Amazon Chime SDK team to prioritize this request
+* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
+* Please go through the existing [issues](https://github.com/aws/amazon-chime-sdk-js/issues) to see if your request has already been recommended
+* Please do not use this template for bugs or support. Each individual repository have a PR template, documentation and resources for troubleshooting (https://github.com/aws/amazon-chime-sdk-js#troubleshooting-and-support).
+* If you are interested in working on this issue or have submitted a pull request, please leave a comment.
+
+# Tell us about your request
+## What do you want us to build?
+Which Amazon Chime SDK or feature area is this request for ?
+Such as: amazon-chime-sdk-js (https://github.com/aws/amazon-chime-sdk-js), amazon-chime-sdk-ios (https://github.com/aws/amazon-chime-sdk-ios), amazon-chime-sdk-android (https://github.com/aws/amazon-chime-sdk-android), a new feature idea
```
[the remark this answers is not in the document]
Comment (vidya-mohan) [reply .github/ISSUE_TEMPLATE/community-request-template.md:20]: good catch

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (vidya-mohan) [reply .github/ISSUE_TEMPLATE/feature_request.md:14]: i should remove this file. we are introducing a community request. This got auto added in the panel

File: .github/ISSUE_TEMPLATE/community-request-template.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+labels: feature-request
+assignees: ''
+
+---
+
+# Community Note
+* Please vote for this issue by adding a 👍 reaction to the issue. This will help the community and maintainers from the Amazon Chime SDK team to prioritize this request.
+* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request.
+* Please go through the existing [issues](https://github.com/aws/amazon-chime-sdk-js/issues) to see if your request has already been recommended.
+* Please do not use this template for bugs or support. Each individual repository has a PR template, documentation and resources for troubleshooting (https://github.com/aws/amazon-chime-sdk-js#troubleshooting-and-support).
```
Comment (devalevenkatesh) [inline .github/ISSUE_TEMPLATE/community-request-template.md:14]: Should this be?
troubleshooting (https://github.com/aws/amazon-chime-sdk-js#troubleshooting-and-support) -> [troubleshooting](https://github.com/aws/amazon-chime-sdk-js#troubleshooting-and-support).

Basically markdown the troubleshooting as it is having an extra space just before the link.

File: .github/ISSUE_TEMPLATE/documentation-request.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 name: Documentation request
 about: Need help using the SDK? Spot an error? Let us know how our documentation can be improved.
 title: ''
-labels: documentation
```
Comment (devalevenkatesh) [inline .github/ISSUE_TEMPLATE/documentation-request.md:5]: Should we keep the "documentation" labels as this is a documentation-request template? Sorry missed this in initial review.

**Review by devalevenkatesh — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/community-request-template.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+* Please vote for this issue by adding a 👍 reaction to the issue. This will help the community and maintainers from the Amazon Chime SDK team to prioritize this request.
+* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request.
+* Please go through the existing [issues](https://github.com/aws/amazon-chime-sdk-js/issues) to see if your request has already been recommended.
+* Please do not use this template for bugs or support. Each individual repository has a PR template, documentation and resources for [troubleshooting](https://github.com/aws/amazon-chime-sdk-js#troubleshooting-and-support).
+* If you are interested in working on this issue or have submitted a pull request, please leave a comment.
+
+# Tell us about your request
+## What do you want us to build?
+Which Amazon Chime SDK or feature area is this request for?
+Such as: [amazon-chime-sdk-js](https://github.com/aws/amazon-chime-sdk-js), [amazon-chime-sdk-ios](https://github.com/aws/amazon-chime-sdk-ios), [amazon-chime-sdk-android](https://github.com/aws/amazon-chime-sdk-android), a new feature idea
```
Comment (devalevenkatesh) [inline .github/ISSUE_TEMPLATE/community-request-template.md:20]: This is still missing a "dot" here. Also, do we want to add React SDK here?

**Review by devalevenkatesh — APPROVED** [review]

**Pull request merged**